lib/discovery.ts

```diff
@@ -59,13 +58,14 @@ function makeMethod (schema, method, context) {
       params: params,
       requiredParams: method.parameterOrder || [],
       pathParams: getPathParams(method.parameters),
-      context: context
+      context: context,
+      mediaUrl: null
```

Comment:
Why is this option added? If this was done only for typescript maybe it should be made optional at the definition site.